src/components/Main/Members/MembersUsersTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import { fade, makeStyles, withStyles, Theme, createStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Checkbox from '@material-ui/core/Checkbox';
+
+import { MemberStateData } from '../../../store/types';
+
+import CSS from 'csstype';
+
+
+const styleHead: CSS.Properties = {
+    fontWeight: 'bold'
+}
+
+export interface MembersUsersTableProps {
+    selectedRows: number[];
+    handleSelectAllClick: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    filteredUsers: MemberStateData[];
+    isSelected: (id: number) => boolean;
+    handleRowClick: (event: React.MouseEvent<unknown>, row: MemberStateData) => void;
+}
+
+export const MembersUsersTable = ({selectedRows, handleSelectAllClick, filteredUsers, isSelected, handleRowClick}: MembersUsersTableProps) => {
+    const classes = useStyles();
+
+    return (
+        <TableContainer component={Paper}>
+            <Table className={classes.table} aria-label="simple table">
```

Comment:
Either remove this specific aria-label, or change it for screen reader a11y:
```suggestion
            <Table className={classes.table} aria-label="members table">
```